# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bluez/hotfix-musl-linux-limits.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bluez-5.66/src/shared/util.c.vanilla	2023-05-27 21:45:29.723925011 +0200
+++ bluez-5.66/src/shared/util.c	2023-05-27 21:48:27.061940542 +0200
@@ -28,6 +28,8 @@
 #include <sys/random.h>
 #endif
 
+#include <linux/limits.h>
+
 #include "src/shared/util.h"
 
 void *util_malloc(size_t size)
